src/System.Private.CoreLib/shared/System/Globalization/StringInfo.cs

```diff
@@ -24,9 +25,11 @@ public StringInfo() : this(string.Empty)
         public StringInfo(string value)
         {
             this.String = value;
+            // workaround: compiler doesn't see this field being initialized through property
```

Comment:
Should we track this workarounds somehow? Or is it enough to just search for the forgiving operator `!` and then fix those cases?

Reply:
so far searching for `!` is almost always suggesting something stinky

Reply:
https://github.com/dotnet/corefx/issues/36427